lib/manager/reducers/projects.js

```diff
@@ -396,21 +458,50 @@ const projects = (state: ProjectsState = defaultState, action: any): ProjectsSta
   }
 }
 
-// TODO: Use this function to get indexes (perhaps set to idx variable?)
-// function getIndexesFromFeed ({state, feedVersion, feedSource, projectId}) {
-//   if (!feedSource && feedVersion) ({feedSource} = feedVersion)
-//   if (!projectId) ({projectId} = feedSource)
-//   const projectIndex = state.all.findIndex(p => p.id === projectId)
-//   const sources = state.all[projectIndex].feedSources || []
-//   const sourceIndex = feedSource && sources.findIndex(s => s.id === feedSource.id)
-//   const versionIndex = feedVersion && sources[sourceIndex].feedVersions
-//     ? sources[sourceIndex].feedVersions.findIndex(v => v.id === feedVersion.id)
-//     : -1
-//   return {
-//     projectIndex,
-//     sourceIndex,
-//     versionIndex
-//   }
-// }
+// helper function to get indexes
+function getIndexesFromFeed ({
```

Comment:
Can you change the comment to a multiline comment? Also, this should maybe move to a util file since this reducer file is already mondo.